addons/point_of_sale/models/pos_order.py

```diff
@@ -946,7 +949,7 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
         payment_ids = self.env["pos.payment"].search([('pos_order_id', 'in', orders.ids)]).ids
         if payment_ids:
             self.env.cr.execute("""
-                SELECT method.name, sum(amount) total
+                SELECT method.name, sum(amount) total, count(DISTINCT payment.pos_order_id)
```

Comment:
@pimodoo Does it bother you to retrieve this COUNT all the time or do we refactor with some _get_fields method?